syntax = "proto3";

service ShenNong {
    rpc Query121(Question) returns (Answers) {}
    rpc Query129(Question) returns(stream Answers) {}
    rpc Query921(stream Question) returns (Answers) {}
    rpc Query929(stream Question) returns(stream Answers) {}
}

message Question {
    int32 QuestionID = 1;
    string Question = 2;
    string Option = 3;
}

message Answers {
    int64 DocID = 1; 
    int32 AnswersID = 2;
    float AnswerScore = 3;
    string Answers = 4; 
}